cmd/ulxly/ulxly.go

```diff
@@ -1308,6 +1321,13 @@ func generateEmptyHashes(height uint8) []common.Hash {
 }
 
 func generateTransactionPayload(ctx context.Context, client *ethclient.Client, ulxlyInputArgBridge string, ulxlyInputArgPvtKey string, ulxlyInputArgGasLimit uint64, ulxlyInputArgDestAddr string, ulxlyInputArgChainID string) (bridgeV2 *ulxly.Ulxly, toAddress common.Address, opts *bind.TransactOpts, err error) {
+	// checks if bridge address has code
+	err = ensureCodePresent(ctx, client, ulxlyInputArgBridge)
+	if err != nil {
+		err = fmt.Errorf("bridge code check err: %w", err)
+		return
+	}
```

Comment:
```suggestion
	if err := ensureCodePresent(ctx, client, ulxlyInputArgBridge); err != nil { 
		return err
	}
```

Reply:
Since `err` is defined in the `return`, this implementation would raise an unnecessary "shadow" warning.
Also, the internal error wrapper helps us understand that the issue is with the bridge contract instead of a random address without code. I thought it would be clearer.
If you don't mind, I'd rather keep the code.

Reply:
ah yes `err` is defined in the return fn, missed that part!